packages/react-dom/src/__tests__/ReactServerRenderingHydration-test.js

```diff
@@ -694,4 +694,16 @@ describe('ReactDOMServerHydration', () => {
       );
     }
   });
+  it('should not warn about style mismatch due to empty values', () => {
+    const domElement = document.createElement('div');
+    const App = () => <div style={{transform: ''}} />;
+
+    const markup = ReactDOMServer.renderToStaticMarkup(<App />);
```

Comment:
This should use `renderToString` for parity with other test. Will check tomorrow if this shouldn't part of the other server integration tests.
```suggestion
    const markup = ReactDOMServer.renderToString(<App />);
```